tests/unit_tests/test_wave_01.py

```diff
@@ -40,16 +40,15 @@ def test_removing_from_inventory_returns_item():
     assert item not in vendor.inventory
     assert result == item
 
-@pytest.mark.skip
+# @pytest.mark.skip
 def test_removing_not_found_is_false():
     item = "item to remove"
     vendor = Vendor(
         inventory=["a", "b", "c"]
     )
 
     result = vendor.remove(item)
-
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
     # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
+    assert item not in vendor.inventory
+    assert result is False
+    assert len(vendor.inventory) == 3
```

Comment:
Since the general intention of this method is to modify the inventory, we want to check this condition as well to make sure that the items that started in the inventory are still there. Adding these assertions would make the test more robust.

```py
    assert "a" in vendor.inventory
    assert "b" in vendor.inventory
    assert "c" in vendor.inventory
```